mask/password.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2020 The Moov Authors
+// Use of this source code is governed by an Apache License
+// license that can be found in the LICENSE file.
+
+package mask
+
+import (
+	"fmt"
+	"strings"
+	"unicode/utf8"
+)
+
+func Password(s string) string {
+	if utf8.RuneCountInString(s) < 3 {
+		return "**" // too short, we can't mask anything
+	} else {
+		// turn 'password' into 'p******d'
+		first, last := s[0:1], s[len(s)-1:]
+		return fmt.Sprintf("%s%s%s", first, strings.Repeat("*", len(s)-2), last)
```

Comment:
Does it matter that this exposes the first and last chars, and the length of the password?

Reply:
This is designed for admin servers of applications and to give operators some level of visibility to ensure the password is correct without exposing the credentials. It's a tradeoff and currently applications would render the entire password.

Reply:
I wonder if we could use a fixed amount of `*`? That'd still be enough for the admin to recognize the first and last chars, but not leak the password length.

Reply:
Yea we can do that. 5 stars?

Reply:
I think that sounds good.